components/server/src/ome/security/SystemTypes.java

```diff
@@ -78,6 +79,8 @@ public boolean isSystemType(Class<?> klass) {
             return true;
         } else if (IGlobal.class.isAssignableFrom(klass)) {
             return true;
+        } else if (Namespace.class.isAssignableFrom(klass)) {
```

Comment:
group is annotatable and global (for example): https://github.com/openmicroscopy/openmicroscopy/blob/v5.1.0-m3/components/model/resources/mappings/meta.ome.xml#L12